### PR TITLE
Feature/page design

### PR DESCRIPTION
**Pull request by jvillalp**
# Description

-Edited 'not interested btn' in recommendations page and added functionality and some styling to 'service providers'.  Appear in all pages except watchlist and onboarding page. 
-merged conflict resolved with master branch

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, but not tested (may need new tests)

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/dashboard/Explore.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ function Explore({
   isSearching,
   notWatchListAction,
 }) {
+  // console.log('this is the movies', movie_id)
```
Comment (ryan-hamblin) [inline src/components/dashboard/Explore.js:46]: Just remove the console logs :)

File: src/components/dashboard/Explore.js [hunk 2]
```diff
@@ -188,6 +189,7 @@ function Explore({
 }
 
 const mapStateToProps = (state) => {
+
```
Comment (ryan-hamblin) [inline src/components/dashboard/Explore.js:192]: Interesting extra space here. I don't know that it's necessary

File: src/components/dashboard/Ratings.js [hunk 3]
```diff
@@ -54,16 +55,18 @@ function Ratings({
...
-                  .toString()
-                  .toLowerCase()
-                  .includes(searchTerm.toLowerCase())
+                .toString()
+                .toLowerCase()
+                .includes(searchTerm.toLowerCase())
               : true
           )
           .map((movie, index) => {
+            // console.log('inratings',movie.movie_id)
```
Comment (ryan-hamblin) [inline src/components/dashboard/Ratings.js:69]: You can just delete this before production :)

File: src/components/dashboard/Ratings.js [hunk 4]
```diff
@@ -72,19 +75,22 @@ function Ratings({
...
                   key={index}
+                  page={"Ratings"}
                   name={movie.primary_title}
                   movie_id={movie.movie_id}
                   year={movie.start_year}
                   rated={movie.rating}
                   description={movie.description}
                   trailer={movie.trailer_url}
                   genres={movie.genres}
+                  // onClick={handleClickProvider}
```
Comment (ryan-hamblin) [inline src/components/dashboard/Ratings.js:86]: Please remove the commented out codes

File: src/components/dashboard/Watchlist.js [hunk 5]
```diff
@@ -83,6 +82,8 @@ function Watchlist({
               : true
           )
           .map((movie, index) => {
+            // console.log('in watchlist',movie.movie_id)
```
Comment (ryan-hamblin) [inline src/components/dashboard/Watchlist.js:85]: Delete :)

File: src/components/dashboard/Watchlist.js [hunk 6]
```diff
@@ -92,11 +93,13 @@ function Watchlist({
                 <MovieCard
                   key={index}
                   name={movie.primary_title}
+                  // movie_id={movie.movie_id}
```
Comment (ryan-hamblin) [inline src/components/dashboard/Watchlist.js:96]: Delete

File: src/components/dashboard/Watchlist.js [hunk 7]
```diff
@@ -92,11 +93,13 @@ function Watchlist({
...
                   year={movie.start_year}
                   trailer={movie.trailer_url}
                   description={movie.description}
                   genres={movie.genres}
                   page="watchlist"
+                  // provider={serviceProvider.data.provider}
```
Comment (ryan-hamblin) [inline src/components/dashboard/Watchlist.js:102]: delete

File: src/store/actions/serviceProviderAction.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
+import axiosWithAuth from "../../utils/axiosWithAuth.js";
+export const FETCHING_SERVICEPROVIDERS_START = "FETCHING_SERVICEPROVIDERS_START";
+export const FETCHING_SERVICEPROVIDERS_SUCCESS = "FETCHING_SERVICEPROVIDERS_SUCCESS";
+export const FETCHING_SERVICEPROVIDERS_FAIL = "FETCHING_SERVICEPROVIDERS_FAIL";
+
+export function serviceProviderAction(id, token, movie_id) {
+    // console.log('movie id', movie_id)
```
Comment (ryan-hamblin) [inline src/store/actions/serviceProviderAction.js:7]: delete

File: src/store/actions/serviceProviderAction.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+        });
+        axiosWithAuth(token)
+            .get(`${id}/service-providers/${movie_id}`)
+            .then(res => {
+                dispatch({
+                    type: FETCHING_SERVICEPROVIDERS_SUCCESS,
+                    payload: res.data,
+                    
+                });
+                // console.log('in action', res.data)
```
Comment (ryan-hamblin) [inline src/store/actions/serviceProviderAction.js:20]: Delete

File: src/store/reducers/serviceProviderReducer.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+const initialState = {
+    serviceProviders: [],
+    isFetching: false,
+    error: ""
+};
+
+
+export const serviceProvider = (state = initialState, action) => {
+    switch (action.type) {
+        // GET service providers
```
Comment (ryan-hamblin) [inline src/store/reducers/serviceProviderReducer.js:16]: these are the right kinds of comments.

**Comment by jvillalp** [comment]
Thank you so much for taking a look Ryan! I will make sure to resolve these comments and get the code cleaned up. Awesome, feedback!

File: src/store/reducers/serviceProviderReducer.js (hunk 10 above)
Comment (jvillalp) [reply src/store/reducers/serviceProviderReducer.js:16]: thank you!

**Pull request merged**